pages.zh/linux/a2disconf.md

```diff
@@ -0,0 +1,12 @@
+# a2disconf
+
+> 在基于 Debian 的操作系统上禁用 Apache 配置文件.
+> 更多信息: <https://manpages.debian.org/buster/apache2/a2disconf.8.en.html>.
+
+- 禁用配置文件:
+
+`sudo a2disconf {{配置文件}}`
```

Comment:
Should we remove the `sudo` from the `a2disconf` english pages? @sbrl @owenvoke

Reply:
I think this is fine. 👍🏻 We have a reference to this [in the docs](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).